core/src/main/resources/org/apache/spark/ui/static/webui.js

```diff
@@ -83,4 +83,10 @@ $(function() {
   collapseTablePageLoad('collapse-aggregated-rdds','aggregated-rdds');
   collapseTablePageLoad('collapse-aggregated-activeBatches','aggregated-activeBatches');
   collapseTablePageLoad('collapse-aggregated-completedBatches','aggregated-completedBatches');
+});
```

Comment:
Nit: add a blank line here.

Reply:
added the blank space as per the review comment. Could you verify